editor/editor_inspector.cpp

```diff
@@ -2028,10 +2054,10 @@ void EditorInspector::_clear() {
 }
 
 void EditorInspector::refresh() {
-	if (refresh_countdown > 0 || changing) {
+	if (changing) {
 		return;
 	}
-	refresh_countdown = EditorSettings::get_singleton()->get("docks/property_editor/auto_refresh_interval");
+	refresh_countdown = 0.01;
```

Comment:
It seems like once there's a `refresh()`, it will force `0.01` and won't ever go back to the configurable value set in the inspector?

Reply:
So far I see:
- `0.2` as default value for the setting
- `0.33` as fallback value if EditorSettings are not available
- `0.01` forced here

Reply:
Actually I should have removed this, will do now since the function makes no sense any longer.